src/api/firebase.js

```diff
@@ -224,6 +225,18 @@ export async function deleteItem(listPath, itemId) {
 	const docRef = doc(db, listPath, 'items', itemId);
 	await deleteDoc(docRef);
 }
+// delete collection path
+export async function deleteList(listPath, email) {
+	const docRef = doc(db, listPath);
+	console.log(docRef);
+	const userDocRef = doc(db, 'users', email);
+
+	await deleteDoc(docRef);
+
+	await updateDoc(userDocRef, {
+		sharedLists: arrayRemove(docRef),
+	});
```

Comment:
Nicely done 💥✨! we don't want the user to have a shared list that does not exist 👍

Reply:
okay changed that